devdoc/tqueue_requirements.md

```diff
@@ -273,3 +274,23 @@ TQUEUE_POP_RESULT TQUEUE_POP(T)(TQUEUE(T) tqueue, T* item, void* pop_function_co
     - **SRS_TQUEUE_01_045: [** `TQUEUE_POP(T)` shall call `dispose_item_function` with `dispose_item_function_context` as `context` and the array entry value whose state was changed to `POPPING` as `item`. **]**
 
   - **SRS_TQUEUE_01_034: [** `TQUEUE_POP(T)` shall set the state to `NOT_USED` by using `interlocked_exchange`, succeed and return `TQUEUE_POP_OK`. **]**
+
+
+### TQUEUE_GET_VOLATILE_COUNT(T)
+```c
+int64_t TQUEUE_GET_VOLATILE_COUNT(T)(TQUEUE(T) tqueue);
+```
+
+`TQUEUE_GET_VOLATILE_COUNT(T)` returns the item count of the queue if available.
```

Comment:
I would put a note here stating the (obvious) fact that the returned value is a point in time value and if the caller needs any synchronization related to the count obtained it is on the caller to lock/use other means of synchronizing.